src/mappers/manfred-to-europass.mapper.ts

```diff
@@ -0,0 +1,52 @@
+import { SkillsPassport } from "@/models/europass"
+import { ManfredAwesomicCV } from "@/models/manfred"
+
+
+export const manfredToEuropass = (manfred: ManfredAwesomicCV): SkillsPassport => {
+    const europass: SkillsPassport = {
+        documentInfo: {
+            documentType: 'ECV',
+            creationDate: new Date(Date.now()),
+            lastUpdateDate: new Date(Date.now()),
+            xsdVersion: 'V3.0',
+            generator: 'EWA',
+            comment: 'Europass CV'
+        },
+        printingPreferences: {
+            type: ''
+        },
+        learnerInfo: {
+            identification:
+            {
```

Comment:
Same for identification but passing the function manfred.aboutMe

Reply:
nested  learnedInfo & identification.
created generate personName function.
should i keep nesting others properties like contactInfo & demographics ?